test/unit/components/uploaderbutton.test.js

```diff
@@ -0,0 +1,28 @@
+/* global chai */
+/* global describe */
+/* global sinon */
+/* global it */
+
+var React = require('react');
+var TestUtils = require('react-addons-test-utils');
+var expect = chai.expect;
+
+import UploaderButton from '../../../app/components/uploaderbutton';
+
+describe('UploaderButton', function () {
+  it('should be a function', function() {
+    expect(UploaderButton).to.be.a('function');
+  });
+
+  describe('render', function() {
+    it('should render without problems', function () {
```

Comment:
I guess we could also test
- the onclick handler works as expected, as making sure the button works feels important
- the text matches what we have given, making sure its not just hardcoded in the `UploaderButton` component